cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/tasks/CassandraDaemonTask.java

```diff
@@ -248,5 +293,26 @@ public CassandraDaemonTask updateId() {
             .build());
     }
 
-
+    @Nullable
+    private static DiscoveryInfo getDiscoveryInfo(
+            Capabilities capabilities, String nodeName, int nativePort) {
+        try {
+            if (!capabilities.supportsNamedVips()) {
+                return null;
+            }
+        } catch (Exception e) {
+            LOGGER.warn("Unable to determine Named VIP support, assuming they are unavailable.", e);
+            return null;
+        }
+        DiscoveryInfo.Builder discoveryBuilder = DiscoveryInfo.newBuilder()
+            .setVisibility(DiscoveryInfo.Visibility.EXTERNAL)
+            .setName(nodeName);
```

Comment:
I think we should namespace the name attribute. For ex: Running multiple Cassandra clusters in the same Mesos cluster.
Ex: https://github.com/verma7/dcos-cassandra-service/commit/f17c31a89eb978afc47d83ed6752fc2fb04c51c9#diff-da60efe1f8e7ad2a82d142a2d97f3d5eR630